v2/concordium/types.proto

```diff
@@ -1393,6 +1396,10 @@ message AccountTransactionEffects {
     BakerConfigured baker_configured = 18;
     // A delegator was configured. The details of what happened are contained in a list of DelegatorEvents.
     DelegationConfigured delegation_configured = 19;
+    // The account has been suspended
+    AccountSuspended account_suspended = 20;
+    // The account has been resumed
+    AccountResumed account_resumed = 21;
```

Comment:
These should be part of `BakerEvent` and not `AccountTransactionEffects` (where they will appear as part of a `BakerConfigured` effect).